backends/tfhe-cuda-backend/cuda/include/aes/aes_utilities.h

```diff
@@ -37,6 +37,7 @@ template <typename Torus> struct int_aes_lut_buffers {
     auto active_streams_and_lut = streams.active_gpu_subset(
         SBOX_MAX_AND_GATES * num_aes_inputs * sbox_parallelism);
     this->and_lut->broadcast_lut(active_streams_and_lut);
+    this->and_lut->setup_gemm_batch_ks_temp_buffers(size_tracker);
```

Comment:
Looks strange to have to set this up manually, but at least we're sure it can only be activated in AES, so let's keep it like this for now.

Reply:
In this way we're sure which ops use GEMM KS instead of relying on some automatic mechanism (solely based on LWE batch size).